dafni_cli/model.py

```diff
@@ -105,27 +102,53 @@ def filter_by_date(self, key: str, date: str) -> bool:
         else:
             raise KeyError("Key should be CREATION or PUBLICATION")
 
-    def output_model_details(self):
+    def output_model_details(self, long=False):
         """Prints relevant model attributes to command line"""
         click.echo(
             "Name: "
             + self.display_name
-            + "     ID: "
+            + TAB_SPACE
+            + "ID: "
             + self.version_id
-            + "     Date: "
-            + self.creation_time.date().strftime(DATE_TIME_FORMAT)
+            + TAB_SPACE
+            + "Date: "
+            + self.creation_time.date().strftime("%B %d %Y")
         )
         click.echo("Summary: " + self.summary)
+        if long:
+            click.echo("Description: ")
+            prose_print(self.description, CONSOLE_WIDTH)
+            click.echo("")
 
     def output_model_metadata(self):
-        """Displays the metadata for the model.
-
-        Args:
-            ctx (context): contains JWT for authentication
-            model_version_id (str): Version ID for the model to display the metadata of
-        """
-        # TODO Implement this
-        pass
+        """Prints the metadata for the model to command line."""
+        click.echo("Name: " + self.display_name)
+        click.echo("Date: " + self.creation_time.strftime("%B %d %Y"))
+        click.echo("Summary: ")
+        click.echo(self.summary)
+        click.echo("Description: ")
+        prose_print(self.description, CONSOLE_WIDTH)
+        click.echo("")
+        if self.metadata.inputs:
+            click.echo("Input Parameters: ")
+            click.echo(self.metadata.format_parameters())
+            click.echo("Input Data Slots: ")
+            click.echo(self.metadata.format_dataslots())
+        if self.metadata.outputs:
+            click.echo("Outputs: ")
+            click.echo(self.metadata.format_outputs())
+
+
+def prose_print(prose: str, width: int):
```

Comment:
This looks useful, might suggest we pull out into some sort of utils file so we can sue this elsewhere